src/injected-js/xhr-helper.ts

```diff
@@ -13,72 +12,37 @@ export default function xhrHelper() {
     // It's important to use fetch when possible because it's needed for
     // getDownloadURL() in Gmail v2: Gmail v2's ServiceWorker in Chrome causes
     // xhr.responseURL to have the wrong value (possibly a Chrome bug).
-    // TODO: This condition will always return true since this function is always defined. Did you mean to call it instead? ts(2774)
-    if (global.fetch as any) {
```

Comment:
`fetch` is supported in every browser we support since #881. The corresponding `else` block for non-fetch support has been removed.